src/core/components/auth/authorize-btn.jsx

```diff
@@ -17,12 +19,18 @@ export default class AuthorizeBtn extends React.Component {
 
     return (
       <div className="auth-wrapper">
-        <button className={isAuthorized ? "btn authorize locked" : "btn authorize unlocked"} onClick={onClick}>
+        <Button
+          className={`sui-btn sui-btn--secondary authorize ${isAuthorized ? "locked" : "unlocked"}`}
```

Comment:
There's a bit of repetition with `sui-btn sui-btn--[moderator]` on _most_ buttons - which I'd prefer to abstract. I was thinking of adding them as default classNames within the `Button` class, with another prop for the moderator - for example: ` mod="primary"`.

Some concerns with this approach is - it limits the `Button` component to these default styles, and also (maybe more importantly) creates a non-familiar interface for a common element. Even so - I'm adding a suggested implementation below - let me know what you think @shockey 

```
// Default Button
<Button
   className={`authorize ${isAuthorized ? "locked" : "unlocked"}`}
   mod="primary"
   onClick={onClick}
 />

// Custom Button
<Button
   className={`authorize ${isAuthorized ? "locked" : "unlocked"}`}
   default=false
   onClick={onClick}
 />
export class Button extends React.Component {

  static propTypes = {
    className: PropTypes.string,
    mod: PropTypes.string,
    default: PropTypes.bool
  }

  static defaultProps = {
    className: "",
    mod: "primary",
    default: true,
  }

  applyDefaultClass = () => this.props.default ? `sui-btn sui-btn--${this.props.mod}` : ""

  render() {
    return (
      <button
        {...this.props}
        className={xclass(this.props.className, "button", this.applyDefaultClass())}
      />
    )
  }
}
```

Reply:
**Change request**: let's go with your implementation @Rody-Kirwan, but invert the `default` prop: instead, let's have an optional `unstyled` prop that, if set, omits the default classes.

Reply:
**Change request**: rename the `authorize` class so that it makes sense w/r/t BEM in the context of this button (perhaps make it a modifier of `sui-btn`? no idea what it's trying to convey)